.gitignore

```
@@ -10,3 +10,14 @@ obj/
 /package/*
 .idea
 trace-agent*
+
+.DS_Store
+node/downloads
+node/content
+node/dev
+node/process_manager/target
+node/src/win-*
+
+## visual studio files
+.vs/
+*.vcxproj.user
```

It's always ok to ignore all `.user` files in VS.

```suggestion
*.user
```